sphinx/ext/intersphinx/_load.py

```diff
@@ -0,0 +1,282 @@
+"""This module contains the code for loading intersphinx inventories."""
+
+from __future__ import annotations
+
+import concurrent.futures
+import functools
+import posixpath
+import time
+from os import path
+from typing import TYPE_CHECKING
+from urllib.parse import urlsplit, urlunsplit
+
+from sphinx.builders.html import INVENTORY_FILENAME
+from sphinx.locale import __
+from sphinx.util import logging, requests
+from sphinx.util.inventory import InventoryFile
+
+from ._shared import InventoryAdapter, InventoryCacheEntry
+
+if TYPE_CHECKING:
+    from typing import IO
+
+    from sphinx.application import Sphinx
+    from sphinx.config import Config
+    from sphinx.util.typing import Inventory
+
+
+logger = logging.getLogger(__name__)
```

Comment:
Ah, I can change the logger name if you want 😅?
although, practically speaking I think it would an extremely niche use case for something to be depending on these logger names (sphinx already suffixes them in `getLogger`)

Reply:
Yes but we never know. I'm not entirely sure that [my extension](https://github.com/picnixz/sphinx-zeta-suppress) would work if you change it (well it would work but you'll need to change the configuration).

So I think it's better if you use the same logger for the whole intersphinx module (or we should make it backward compatible).

Reply:
fixed